ecosystem/sep-0010.md

```diff
@@ -82,6 +84,7 @@ Name      | Type          | Description
 ----------|---------------|------------
 `account` | `G...` string | The stellar account that the wallet wishes to authenticate with the server
 `home_domain` | string | (optional) The [fully qualified domain name](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of the service requiring authentication. SEP-10 signing servers used to generate tokens for multiple different web services can use this parameter to create valid tokens for each service. If not provided by the client, a default should be used.
+`memo` | integer | (optional) The 64-bit unsigned integer used to identify the user in the case of a pooled `account`. Note that the `memo_type` field is not needed because all values are of type `MEMO_ID`.
```

Comment:
See note further down about leveraging the `M` address format to encode the memo and account together. I think we could leverage that format for just SEP-10.